lib/msf/core/post/linux/kernel.rb

```diff
@@ -0,0 +1,69 @@
+# -*- coding: binary -*-
+require 'msf/core/post/common'
+
+module Msf
+class Post
+module Linux
+module Kernel
+  include ::Msf::Post::Common
+
+  #
+  # Returns the kernel version
+  #
+  # @return [String]
+  #
+  def kernel_version
```

Comment:
`uname -r` is used a few times in the framework.

`uname -v` is never used.

It might make more sense to stick with the GNU nomenclature and use both.

```ruby
  #
  # Returns the kernel version
  #
  # @return [String]
  #
  def kernel_version
    cmd_exec('uname -v').to_s
  rescue
    raise 'Could not determine kernel version'
  end

  #
  # Returns the kernel release
  #
  # @return [String]
  #
  def kernel_release
    cmd_exec('uname -r').to_s
  rescue
    raise 'Could not determine kernel version'
  end
```